test/BlockManager.js

```diff
@@ -446,19 +494,11 @@ describe('BlockManager', function () {
         iteration,
         biggestStakerId);
       const proposedBlock = await blockManager.proposedBlocks(epoch, 0);
-      assertBNEqual(proposedBlock.proposerId, toBigNumber('4'), 'incorrect proposalID');
+      assertBNEqual(proposedBlock.proposerId, toBigNumber('5'), 'incorrect proposalID');
 
       await mineToNextState();
 
       const sortedVotes = [toBigNumber('20000')];
-      const {
```

Comment:
Hey @GaurShekhar, Why we are removing this

Reply:
Initially we were calculating the median and everything as we were comparing it with the afterDisputeReset but in this test case we are resetting the values so I have modified the test case to check the equality to it's resetting value rather than the calculated value.